hardware/arduino/avr/boards.txt

```diff
@@ -288,8 +288,8 @@ leonardo.bootloader.low_fuses=0xff
 leonardo.bootloader.high_fuses=0xd8
 leonardo.bootloader.extended_fuses=0xcb
 leonardo.bootloader.file=caterina/Caterina-Leonardo.hex
-leonardo.bootloader.unlock_bits=0x3F
-leonardo.bootloader.lock_bits=0x2F
+leonardo.bootloader.unlock_bits=0xFF
+leonardo.bootloader.lock_bits=0xCF
```

Comment:
I could certainly be wrong on this but shouldn't this(and all other ATmega32U4 entries) be 0xEF? http://eleccelerator.com/fusecalc/fusecalc.php?chip=atmega32u4&LOW=FF&HIGH=D8&EXTENDED=CB&LOCKBIT=2F shows the original lock bits setting is:
- Application protection mode 1
- Boot Loader protection mode 2
- Mode 1: No memory lock features enabled
http://eleccelerator.com/fusecalc/fusecalc.php?chip=atmega32u4&LOW=FF&HIGH=D8&EXTENDED=CB&LOCKBIT=CF shows 0xCF is:
- Application protection mode 1
- Boot Loader protection mode 3
- Mode 1: No memory lock features enabled
http://eleccelerator.com/fusecalc/fusecalc.php?chip=atmega32u4&LOW=FF&HIGH=D8&EXTENDED=CB&LOCKBIT=EF shows 0xEF is the same as the original setting:
- Application protection mode 1
- Boot Loader protection mode 2
- Mode 1: No memory lock features enabled